### DISCUSSION TITLE
Emscripten versions (3.1.34 vs. 3.1.28)

### DISCUSSION THREAD

**Discussion by craigsapp** [rism-digital/verovio#3349] [Q&A] [answered]
I was updating Emscripten emsdk on a computer and I notice that verovio has problems related to an undefined variable/function when running in the latest version (3.1.34), so I switched to the one I am currently using (3.1.28).  Is that a known problem (@WolfgangDrescher)?   I have seen some discussion probably related to that in the past few months in the verovio issues/discussion

Or is there new Javascript code that is necessary to load verovio with emsdk 3.1.34?

**Comment by WolfgangDrescher** [comment, marked as answer]
I just updated to `emcc` v3.1.34 and compiled with `buildNpmPackage` without any problems. Also using `verovio-toolkit-wasm.js` worked as expected. What error message do you get? In the JS console or during compilation?
